src/tasks/center_out_reach/metrics.py

```diff
@@ -232,6 +233,7 @@ def plot_metrics(self, targets):
         Args:
             targets: List of target positions in pixels.
         """
+        pygame.display.init()
```

Comment:
Is this needed?

Reply:
I don't think it is. The other changes are, though.

Reply:
In my tests, it only works well with this command.
The main window with the main task is destroyed by a `pygame.display.quit()` call. So when the metrics window shows, the display module should be started again with the `init()` call.
Although the pyplot library manages to show the window without that `init()`, it doesn't do it well because the metrics window shows and gets frozen.